tests/baselines/reference/numericEnumMappedType.symbols

```diff
@@ -49,7 +49,7 @@ const e2: E2 = E2.ONE;
 
 b1[1] = "a";
 >b1 : Symbol(b1, Decl(numericEnumMappedType.ts, 8, 5))
->1 : Symbol(1)

```

Comment:
...wat? So all enum reverse mapping symbol names are a computed name of the enum member now? Is that...good...?